packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -60,6 +61,14 @@ function processRequest(
   const type = request.type === RequestType.XHR ? ResourceType.XHR : ResourceType.FETCH
 
   const matchingTiming = matchRequestTiming(request)
+  if (!matchingTiming) {
+    addTelemetryDebug('Missing PerformanceResourceTiming', {
+      duration: request.duration,
+      resolveDuration: request.resolveDuration,
+      status: request.status,
+      requestType: request.type === RequestType.XHR ? 'xhr' : 'fetch',
+    })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this might yield a *lot* of logs. Maybe log it only the first time we got a missing timing?